src/components/about/About.js

```diff
@@ -14,7 +15,7 @@ const About = () => (
       <WavesBackground>
         <Section.Stack style={styles.content} grow justifyContent="space-between">
           <Section.Text fontSize={22} color="darkBlue">
-            Welcome to
+            <Trans>Welcome to</Trans>
```

Comment:
P.S. i see you're using also `t` macro. 
As i understood it wraps any string as the i18n label and is processed by the webpack ?
If yes let's use it everywhere e.g.

```javascript
import { t } from '@lingui/macro'

<Section.Text fontSize={22} color="darkBlue">
  {t`WelcomeTo`}
  // or if this is possible to configure
  t`WelcomeTo`
</Section.Text>
```

Reply:
There are some cases where the `t` macro cannot replace `<Trans>`. Lingui gives this example:
```
<Trans>
   Any component is <strong className="green">valid</strong> here, even <Link>custom</Link> ones.
</Trans>
```
where the resulting translation string in would be: `Any component is <0>valid</0> here, even <1>custom</1> ones.`
That allows translators to understand context when translating the texts.
I will replace `Trans` for `t` where it's possible.

Reply:
we have to wrap just the raw strings doesn't having any jsx inside so this case won't be possible
please read the docs about gettext library to understand what king of label wrappers i would have

Reply:
I looked at the documentation again and there is no way to replace `Trans` for `t` with every feature it has. Lingui developers left a full explanation for that here: https://github.com/lingui/js-lingui/issues/15
Here are some examples of generated translation strings in the GoodDAPP text that would cause translation problems:
```
By signing up and entering, you are accepting our\n<1>Terms of Use</1>and <2>Privacy Policy</2>
please save your 12-word pass phrase\n<1>and keep it in a secure location\nso you can recover your wallet anytime</1>
By clicking the 'Create a wallet' button,\nyou are accepting our\n<1>Terms of Use</1>and <2>Privacy Policy</2>
please save your 12-word pass phrase\n<1>and keep it in a secure location\nso you can recover your wallet anytime</1>
<0>Or contact </0><1>support</1>
```
In those cases, should the translation wrapping be left as is with `<Trans>` or turned in to many different separate strings using only `t`, sacrificing ease of use from Crowdin?

Reply:
@alecsantos96 

1. let's use <Trans> for the big text blocks only
2. let's try to shortify its name e.g. 

```javascript
import { Trans as _ } from '@lingui/macro'

const Component = () => (
  <_>
    Any component is <strong className="green">valid</strong> here, even <Link>custom</Link> ones.
  </_>
)
```
(i've checked - it works)

Reply:
@johnsmith-gooddollar @alecsantos96 I dont think shortening Trans is important. lets close this

Reply:
@sirpy that's just a global search replace. it could be done in 5min